draft-ietf-tls-tls13.md

```diff
@@ -3056,9 +3063,29 @@ This section describes protocol types and constants.
 
 ## The Cipher Suite
 
-The following values define the cipher suite codes used in the ClientHello and
-ServerHello messages.
-A cipher suite defines a cipher specification supported in TLS.
+A cipher suite defines a cipher specification supported in TLS and negotiated
+via hello messages in the TLS handshake.
+Cipher suites follow a general naming convention:
+
+       CipherSuite TLS_KEA_SIGN_WITH_CIPHER_HASH = VALUE;
```

Comment:
I could explicitly mention that the "TLS" & "WITH" parts are just static parts of the naming. That could make it more clear, as well.